packages/test-utils/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@terra-ui-packages/test-utils",
   "version": "0.0.2",
-  "private": false,
   "type": "module",
   "module": "./src/index.mjs",
```

Comment:
feels like this package should have a small README to call out that it is not a standard package setup and should not be modeled as such...

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4116/commits/a95822f25ce85cb2733af8e53b84647a6a7b1cff.